### ISSUE TITLE
Advise

### ISSUE THREAD

**Issue by PeterJD** [dzimchuk/book-fast-service-fabric#15]
Local.xml` that is used in local deployment profiles.

Does the above setting enable application to be deployed locally, not via Azure?
if yes where to place file?

Thanks for good code and documentation

With the incredible development of Blazor, would it be possible to implement here?

Kind regards
Peter

**Comment by dzimchuk** [comment]
@PeterJD Yes, Local.xml is the configuration file used by local deployments. The profiles expect to find it in the ApplicationParameters folder (https://github.com/dzimchuk/book-fast-service-fabric/tree/master/BookFast/ApplicationParameters).

I'm also very excited about the development of Blazor and the alternative approach to develop SPAs that it offers. It's on my list to look at someday in the future.

**Comment by PeterJD** [comment]
Hi 
Thanks , don't leave Blazor to much in the future, sooner the better?

Kind regards
Peter
UK, Nottingham
